<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionReclamation</class>
 <widget class="QDialog" name="GestionReclamation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1111</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Ajouter</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>230</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Description :</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>581</width>
       <height>431</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Ajout</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>175</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Date_reclamation :</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>180</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sujet du reclamation :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ref_reclamation :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>230</y>
        <width>381</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_21">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>390</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ID_reservation:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_7">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_15">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>140</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <zorder>groupBox_6</zorder>
    <zorder>label_7</zorder>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Afficher</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1111</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Afficher</string>
     </property>
     <widget class="QTableView" name="tabrec">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>461</width>
        <height>192</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>30</y>
        <width>551</width>
        <height>441</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Modifier</string>
      </property>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>380</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Franklin Gothic Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_28">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>380</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Franklin Gothic Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Description :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Date_reclamation :</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_2">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>170</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ref_reclamation :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_14">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>50</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>220</y>
         <width>381</width>
         <height>141</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>211</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sujet du reclamation :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>201</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ID_reservation:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_8">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>90</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="lineEdit_19">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>160</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>290</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Recherche:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>330</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>ref_rec</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>date_rec</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>sujet</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>description</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_18">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>400</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushPDF1">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>400</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <zorder>groupBox_5</zorder>
     <zorder>tabrec</zorder>
     <zorder>lineEdit_19</zorder>
     <zorder>label_9</zorder>
     <zorder>comboBox_6</zorder>
     <zorder>pushButton_18</zorder>
     <zorder>lineEdit_7</zorder>
     <zorder>pushPDF1</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
